Gemfile

```diff
@@ -17,6 +17,7 @@ gem 'i18n-js', '3.5.1' # A library to provide the I18n translations on the Javas
 # Authentications & Authorizations
 gem 'devise' # Authentication solution for Rails with Warden
 gem 'pundit' # Minimal authorization through OO design and pure Ruby classes
+gem 'doorkeeper' # Awesome OAuth 2 provider for your Rails / Grape app
```

Comment:
I think this only needed for API, usually we handle all API related changes in a separate PR(ex PR title: [API] As user ...) and  separate task (ex: [API] As user ...). Usully we don't do mix changes API and Web in a single PR. For upcoming PR you can separate changes in a separate PR 👍  (API related changes will be in separate API PR, Web related changes will be in separate Web PR).

Reply:
Well noted, I'll be more careful for the next ones! ;-) Thanks